modules/test/playwright/tests/calendar-web/calendarEvent.spec.ts

```diff
@@ -72,10 +73,52 @@ test('can create all-day calendar event with different time zone', async ({
 	await expect(endTime).toEqual(startTime);
 });
 
+test('can update an event with recurrence', async ({calendarWidgetPage}) => {
+	await calendarWidgetPage.fillEventWithRecurrenceUntilDate();
+
+	await expect(
+		calendarWidgetPage.page
+			.frameLocator('iframe')
+			.getByRole('textbox', {name: 'mm/dd/yyyy'})
+	).toBeEnabled();
+
+	await calendarWidgetPage.page
+		.frameLocator('iframe')
+		.getByRole('button', {name: 'Done'})
+		.click();
+
+	await calendarWidgetPage.publishEvent();
+
+	await calendarWidgetPage.page
+		.frameLocator('iframe')
+		.getByLabel('Title', {exact: true})
+		.click();
+
+	await calendarWidgetPage.page
+		.frameLocator('iframe')
+		.getByLabel('Title', {exact: true})
+		.fill(getRandomString());
+
+	await calendarWidgetPage.page
+		.frameLocator('iframe')
+		.getByRole('button', {exact: true, name: 'Publish'})
+		.click();
+
+	await calendarWidgetPage.page
+		.frameLocator('iframe')
+		.getByRole('button', {name: 'Following Events'})
```

Comment:
You should cover both 'Following Events' and also 'Entire Series' case. Can you update the test to do it so?